### PR TITLE
RD-1055: update balance definition.

### PR DESCRIPTION
**Pull request by VKCR**

### CODE REVIEW

File: src/proto/index.js.flow [hunk 1]
```diff
@@ -357,7 +357,22 @@ declare export class Balance {
...
+        updatedAtMs: number,
+        typedBalances: Array<TypedBalance>,
+    }): Balance;
+    toJSON(): Object;
+}
+declare export class TypedBalance {
+    type: string;
+    amount: Money;
+    updatedAtMs: number;
+    static create({type: string, amount: Money, updatedAtMs: number}): Balance;
```
Comment (sam-token) [inline src/proto/index.js.flow]: should this return Balance or TypedBalance?

Comment (VKCR) [reply src/proto/index.js.flow]: oops. Fixed.

File: src/proto/index.js.flow [hunk 2]
```diff
@@ -357,7 +357,22 @@ declare export class Balance {
...
+    static create({
+        accountId: string,
+        current: Money,
+        available: Money,
+        updatedAtMs: number,
+        typedBalances: Array<TypedBalance>,
+    }): Balance;
+    toJSON(): Object;
+}
+declare export class TypedBalance {
```
Comment (zixunjian) [inline src/proto/index.js.flow]: is there a need for it to be an exported class (would the client ever need to create a TypedBalance)? If not we can use `type TypedBalance = {...}` and get rid of create() and toJSON()

Comment (VKCR) [reply src/proto/index.js.flow]: done

**Pull request merged**